### DISCUSSION TITLE
How to custom field type?

### DISCUSSION THREAD

**Discussion by ouesiengi** [directus/directus#12674] [Archive]
<img width="1020" alt="D2256C56-46B9-450D-8B45-C8AAF1F0634D" src="https://user-images.githubusercontent.com/52437407/162704560-fa178978-7f2b-4350-8658-35dcf3fadef8.png">

**Comment by azrikahar** [comment]
You can create custom interfaces (the new field choices in your screenshot above) 👍 Link to docs: https://docs.directus.io/extensions/interfaces/

You can also refer to https://docs.directus.io/extensions/creating-extensions/ and get started with a boilerplate.
